complex/src/lib.rs

```diff
@@ -758,6 +810,15 @@ impl<T: Clone + Num> Div<T> for Complex<T> {
     }
 }
 
+impl<T: Clone + Num + PartialOrd> Rem<T> for Complex<T> {
```

Comment:
I think now this doesn't need `PartialOrd`, right?  And we can revert the related macro changes.

Reply:
Yes, good point.